provider/cmd/app/app.go

```diff
@@ -117,6 +120,9 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&o.Libvirt.Qcow2Type, "qcow2-type", qcow2.Default(), fmt.Sprintf("qcow2 implementation to use. Available: %v", qcow2.Available()))
 
+	fs.DurationVar(&o.VMGracefulShutdownTimeout, "vm-graceful-shutdown-timeout", 5*time.Minute, "Duration to wait for the VM to gracefully shut down. If the VM does not shut down within this period, it will be forcibly terminated.")
```

Comment:
what do you think about gc-vm-graceful-shutdown-period/timeout. Mainly i want to know gc-resync-interval and this argument have some relationship.

Reply:
I would prefer to mention the link in the description. Something like: `Duration to wait for the VM to gracefully shut down. If the VM does not shut down within this period, it will be forcibly destroyed by garbage collector`

Reply:
Added it to the description, as well as the name is changed.